### PR TITLE
Feature/us 400386

### PR DESCRIPTION
**Pull request by saikarthik528**
As part of hazelcast upgrade from version 3 to version 4, platform expects a prconfig flag (cluster/hazelcast/v4/enabled), to be set as true, which denotes that version 4 is enabled by default for any installations from pega-helm-charts.

### CODE REVIEW

**Review by MadhuriArugula — APPROVED** [review]

**Review by kishorv10 — APPROVED** [review]

File: charts/pega/config/deploy/prconfig.xml [hunk 1]
```diff
@@ -15,4 +15,6 @@
         for this node.  This is useful for settings which are specific to this node and should not be shared by multiple nodes on this
         system. -->
 
+        <env name="prconfig/cluster/hazelcast/v4/enabled" value="true" type="java.lang.String"/>
+
```
Comment (Heshma) [inline charts/pega/config/deploy/prconfig.xml:19]: @saikarthik528 Could you please add a short comment on the usage of this setting?

File: charts/pega/charts/installer/config/prconfig.xml.tmpl [hunk 2]
```diff
@@ -45,4 +45,5 @@
   <env name="install/parallel/threadcount" value="-1" />
   <env name="install/parallel/index/threadcount" value="-1" />
   <env name="compiler/javasourcelevel" value="1.7" />
+  <env name="prconfig/cluster/hazelcast/v4/enabled" value="true" type="java.lang.String"/>
```
Comment (PraneethaYanala) [inline charts/pega/charts/installer/config/prconfig.xml.tmpl:48]: If we are setting in prconfig.xml then we should not add prconfig to config name it should be <env name="cluster/hazelcast/v4/enabled" value="true" type="java.lang.String"/>, same applies for all the prconfig files

Comment (PraneethaYanala) [inline charts/pega/charts/installer/config/prconfig.xml.tmpl:48]: If we are setting in prconfig.xml then we should not add prconfig to config name it should be <env name="cluster/hazelcast/v4/enabled" value="true" type="java.lang.String"/>, same applies for all the prconfig files

Comment (saikarthik528) [reply charts/pega/charts/installer/config/prconfig.xml.tmpl]: Addressed this in the latest push

File: charts/pega/config/deploy/prconfig.xml (hunk 1 above)
Comment (saikarthik528) [reply charts/pega/config/deploy/prconfig.xml:20]: added comment about usage

**Pull request merged**